src/service-runner-config.json

```diff
@@ -14,9 +14,9 @@
             "start": [
               "--classic",
               "--datadir",
-              "${SERVICE_DIR}/multi-geth/1.9.9/classic",
+              "${SERVICE_DIR}/core-geth/1.11.2/classic",
```

Comment:
Same question as above about using a new dir and leaving one disused.

Reply:
It does leave a dangling path. There's always been a discussion of garbage collection I believe Zack put up an issue some time ago. We have the information to garbage collect, but it's uncertain if we should leave that in the hands of the user.  Service runner atm doesn't allow for removal without access.

I'm probably going to punt this issue until we have a better story around shared config . Atm the deploy step will blast this data.

Reply:
The path to garbage collection is in the manifest.json file